docs-developer/references/release_process.rst

```diff
@@ -144,6 +144,11 @@ Copy the entries from the changelog into Github's "Release notes".
     chronology will break. Do not add tags in feature branches or in the master
     branch. You can add tags for pre-releases in ``develop``, for releases that don't yet have a release branch.
 
+.. warning:: Tagging is known to break after rebasing, so in case you rebase
+    a branch after tagging it, delete the tag and add it again. Basically,
+    ``git describe --tags`` detects the closest tag, but after a rebase, its
```

Comment:
Friends don't let friends tag and rebase.

Reply:
> so in case you rebase a branch after tagging it, delete the tag and add it again

maybe this should say 

> so never rebase a published branch

:)

Reply:
@indirectlylit I think -- but I don't wanna say 100% sure -- that this is something that happens when you rebase and you're supposed to get some tag into your commit history before your current commits.. then somehow the distance logic that `git describe` applies isn't intuitive.. I think we've seen a lot of this in PRs.

Reply:
@indirectlylit but of course it should not read as if we can just delete and re-add existing tags.

Reply:
Let me know if it does, then it should be changed!

Reply:
Is it better to be clear that **release** tags should only be applied to published **release** branches (and preferably `master`) ?

Reply:
I think it's fine for now!